opentelemetry-api/src/opentelemetry/_events/__init__.py

```diff
@@ -0,0 +1,196 @@
+from abc import ABC, abstractmethod
+from logging import getLogger
+from os import environ
+from typing import Any, Optional, cast
+
+from opentelemetry._logs import LogRecord
+from opentelemetry._logs.severity import SeverityNumber
+from opentelemetry.environment_variables import (
+    _OTEL_PYTHON_EVENT_LOGGER_PROVIDER,
+)
+from opentelemetry.trace.span import TraceFlags
+from opentelemetry.util._once import Once
+from opentelemetry.util._providers import _load_provider
+from opentelemetry.util.types import Attributes
+
+_logger = getLogger(__name__)
+
+
+class Event(LogRecord):
+
+    def __init__(
+        self,
+        name: str,
+        timestamp: Optional[int] = None,
+        trace_id: Optional[int] = None,
+        span_id: Optional[int] = None,
+        trace_flags: Optional["TraceFlags"] = None,
```

Comment:
I noticed in javascript and java, there's a context param vs certain things inside it. Is it intentional to deconstruct the context like this?

https://github.com/open-telemetry/opentelemetry-js/blob/d91dbe1b1decea41cfce55213b545e45e9ef67ef/experimental/packages/api-events/src/types/Event.ts#L50
https://github.com/open-telemetry/opentelemetry-java/blob/c43835970c1165a6667808125581397ed27652fc/api/incubator/src/main/java/io/opentelemetry/api/incubator/events/EventBuilder.java#L137

Reply:
@codefromthecrypt 

Might be just a stylistic choice. Java does tend to encapsulate fields inside classes and Python tends to have the explicit params.

Reply:
cool mainly was curious if we are missing anything from the context, such as tracestate or whatever, as a consequence, if so maybe we need a unit test to show we can recreate it? This is drive-by advice from someone who doesn't primarily work in python, so please don't anchor too much.